core/src/main/java/io/questdb/cutlass/auth/AuthUtils.java

```diff
@@ -73,4 +168,14 @@ public static PublicKey toPublicKey(CharSequence encodedX, CharSequence encodedY
             throw new IllegalArgumentException("Failed to decode " + encodedX + "," + encodedY, ex);
         }
     }
+
```

Comment:
I think :
- n should be set to `signatureRaw.limit()` or 
- loop condition should be  ```i - position() < n```

Reply:
yes, of course.
🤦